pyproject.toml

```diff
@@ -13,30 +13,32 @@ classifiers= [
     "Topic :: Scientific/Engineering",
     "Topic :: Multimedia :: Video",
     "Topic :: Multimedia :: Graphics",
+	"Programming Language :: Python :: 3",
     "Programming Language :: Python :: 3.6",
     "Programming Language :: Python :: 3.7",
     "Programming Language :: Python :: 3.8",
+	"Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3 :: Only",
     "Natural Language :: English",
     ]
-exclude = ["scripts/","logo/","readme-assets/"]
+exclude = ["scripts/","logo/"]
 packages = [
     { include = "manim" },
 ]
 
 [tool.poetry.dependencies]
 python = "^3.6.2"
-colour = "*"
+colour = "^0.1.5"
 numpy = "*"
 Pillow = "*"
 scipy = "*"
 tqdm = "*"
 pydub = "*"
 pygments = "*"
-rich = "^6.0"
+rich = "^9.0"
```

Comment:
did this cause the doctest to fail?

Reply:
I don't think so. I was able to reproduce the weird logging behavior that the[ test cases are reporting](https://github.com/ManimCommunity/manim/pull/1016/checks?check_run_id=1872002075#step:14:58) in the `Got:` in my click branch (which still has `rich = ^6.0`) by entering in the interpreter:
```python
>>> from manim import *
>>> SingleStringMathTex('Test')
```
Unfortunately, I haven't been able to do reproduce  since trying on your branch/master.

Reply:
Don't know what really happens but it looks super weird and needs some investigation.